includes/object-cache.php

```diff
@@ -1897,7 +1897,10 @@ function ( $key ) use ( $derived_keys ) {
         } catch ( Exception $exception ) {
             $this->handle_exception( $exception );
 
-            $cache = array_fill( 0, count( $derived_keys ) - 1, false );
```

Comment:
fwiw: this currently drops any cache hits from the internal cache set in https://github.com/rhubarbgroup/redis-cache/blob/c70718a9bfa3c87a154f6b6a38125c4275cd1b34/includes/object-cache.php#L1857-L1867.